pkg/model/iam/iam_builder.go

```diff
@@ -293,11 +293,13 @@ func addRoute53ListHostedZonesPermission(p *IAMPolicy) {
 }
 
 // IAMPrefix returns the prefix for AWS ARNs in the current region, for use with IAM
-// it is arn:aws everywhere but in cn-north, where it is arn:aws-cn
+// it is arn:aws everywhere but in cn-north and us-gov-west-1
```

Comment:
It is the only region currently. 

This patch allowed me to create a cluster there, but I have hit a couple of other minor issues and several limitations so far. For example, GovCloud doesn't allow NAT Gateways or provide Route 53. Is there a good place for me to create a document describing use of kops in GovCloud? It might save others some time.

Also, should I create individual tickets for each thing I encounter or a broader GovCloud ticket?